crates/matrix-sdk-crypto/src/types/events/utd_cause.rs

```diff
@@ -180,6 +212,24 @@ mod tests {
         );
     }
 
+    #[test]
+    fn if_reason_is_not_missing_key_we_guess_unknown_even_if_membership_is_leave_we_guess_membership(
```

Comment:
We have a convention (which I personally don't like, but do comply with) of test names starting with `test_`

Reply:
For me, consistency within a file is more important than the convention for new tests

Reply:
Then please prefix all the tests names with `test_`. This makes it easier to distinguish test functions from non-test functions in LSP searches.